cli/cmd/dashboard.go

```diff
@@ -97,6 +94,6 @@ func init() {
 
 	// This is identical to what `kubectl proxy --help` reports, `--port 0`
 	// indicates a random port.
-	dashboardCmd.PersistentFlags().IntVarP(&dashboardProxyPort, "port", "p", 8001, "The port on which to run the proxy. Set to 0 to pick a random port.")
+	dashboardCmd.PersistentFlags().IntVarP(&dashboardProxyPort, "port", "p", 0, "The port on which to run the proxy. Set to 0 to pick a random port.")
```

Comment:
NIT: Since the default is now 0, I would change this message to "When set to 0, a random port will be used."